docs/sources/http_api/admin.md

```diff
@@ -282,6 +282,54 @@ Content-Type: application/json
 }
 ```
 
+## Grafana Usage Report preview
+
+`GET /api/admin/usage-report-preview`
```

Comment:
I am a bit hesitant about the naming of the endpoint. As far as I understand this is the exposing the [view of admin server stats](https://grafana.com/docs/grafana/latest/administration/view-server/view-server-stats/), given that we have features like "Usage Insights", I am a bit worried that "usage report preview" can be confusing for users.

I know this is not related just to the documentation, and is more about the API itself, however wanted to open the discussion as if we decide to change, my hope is that it wouldn't be too much of a problem.

Reply:
This is actually linked to https://github.com/grafana/grafana/pull/43899 which implements `usage-report-preview` which gives a preview of Usage insights for users to know what they are sending. 

If I correctly understand your point, I'm very happy that it immediately made you think of Usage Insights

Reply:
Thanks for clarifying. Makes sense in that case. 

Does it make sense also to reference this endpoint somewhere in the main usage insights docs? for example in the index page - https://grafana.com/docs/grafana/latest/enterprise/usage-insights/

Reply:
> This is actually linked to #43899 which implements usage-report-preview which gives a preview of Usage insights for users to know what they are sending.

This is afaict different from usage insights, usage insights is for frontend performance and is sent to the backend with details about load times. https://grafana.com/docs/grafana/latest/enterprise/usage-insights/

I'm not sure if this has a reasonable name, since it's not been userfacing.

Reply:
Just had a second look at the changes, as @sakjur pointed above, this is indeed not the usage insights. What is known as [Usage Insights](https://grafana.com/docs/grafana/latest/enterprise/usage-insights/), is a Grafana Enterprise feature, whereas the data which the new API endpoint is surfacing is collected for analytical purposes by the Grafana backend. 

Would `/api/admin/usage-stats-report-preview` be a better alternative, as the service itself is called [usagestats](https://github.com/grafana/grafana/blob/main/pkg/infra/usagestats/service/usage_stats.go#L22). 

Also as I mentioned in my [comment here](https://github.com/grafana/grafana/pull/43899/files#r788528361), we would need to look into adding a fine-grained permission to the endpoint.

Reply:
That sounds right

Reply:
👍 I am fine staying with the current naming and calling it usage report. 

@Jguer last thing could check please my comment [here](https://github.com/grafana/grafana/pull/43995#discussion_r788532807). Just want to make sure we are all good on fine-grained access control side. Thanks.